app/models/server/models/LivechatDepartmentAgents.js

```diff
@@ -30,6 +31,7 @@ export class LivechatDepartmentAgents extends Base {
 		}, {
 			$set: {
 				username: agent.username,
+				departmentEnabled: agent.departmentEnabled,
```

Comment:
This implementation is OK, but I found some calls to the `saveAgents` method that are passing the `departmentEnabled`, like [here](https://github.com/RocketChat/Rocket.Chat/blob/develop/app/models/server/models/LivechatDepartment.js#L84).
Is that supposed to be okay?